index.html

```diff
@@ -6581,8 +6581,8 @@ <h3>Executing Script</h3>
     <li><p>Return <a>success</a> with data <var>reference</var>.
   </ol>
 
- <dt>has an <a>own property</a> named "<code>toJSON</code>" that is
-  a <a>Function</a>
+ <dt>has an <a>own property</a> or <a>inherited property</a> named
```

Comment:
```suggestion
 <dt>has a property</a> named
```

Given that it doesn't matter if it's an own or inherited property we potentially could just use this term?